### ISSUE TITLE
Plot issue

### ISSUE THREAD

**Issue by bardkin** [ForgeEssentials/ForgeEssentials#2119]
Forge Essentials 1.7.10-1.3.1-Beta1-server
I am looking to migrate a world to Forge essentials so I made a copy of it and put it in a new folder and profile. So far so good.

I created a test area called Plot1
( /area define plot1 )
now I want to delete it.. it was just a test, after all.
( /area delete plot1 )

Game-"Plot1" has does not exist!

so i do /area list and get:

#5: WORLD_0_Plot1

I tried the names "#5: WORLD_0_Plot1" and "#5:"

So- what do I need to do to remove the test plot? Can I go to a config or something and remove it there?
Also- I tried to do 
/area entry <message> with it and it doesn't work either. same error.

Anything I can do?
Thanks!

**Comment by spacebuilder2020** [comment]
You can delete it manually by going into the FEData/permissions.json file.  I don't know much about plots cause I use mytown2 for Greif Prevention on my server and don't even mess with plots.

**Comment by bardkin** [comment]
ok, thanks, I'll try that. I will also look into MyTown, but im just trying to set up a single world for me and my kids to play so I dont want anything too involved or that's overkill. Really, all I want to do is protect the towns where the quests are setup- no creepers and such..
Thanks again though!

**Comment by spacebuilder2020** [comment]
Just do /gamerule mobGriefing false if you just want to prevent creepers from destroying stuff.

The only reason for plots is to prevent other players from greifing your stuff, but if it is just your kids you should not have to worry about that.

**Comment by bardkin** [comment]
Well, let me ask you this- if there is a way ( or better way )
What I am doing is creating a world that has several towns (Mo' Villages) and quests (Custom NPC's)
I want to prevent creepers from blowing up places in these towns, but I don't want to disable them world wide.
What I need to do is maybe select a region encompassing the town and then flag it for no creepers or no explosions, or even no kids building stuff.
I was looking into using world guard but it 
A) needs Bukkit and 
B) you can't migrate a world into it- as far as I can tell..

So what would be the best solution to accomplish this or does it require building a whole new world from scratch? - in which case I would go with a different set of mods..

**Comment by spacebuilder2020** [comment]
You can use KCauldron.  However, mytown2 should be easily sufficient and that is forge based.

**Comment by bardkin** [comment]
Ok. i have it installed into a copy of my world so I can monkey with it and try it out. Reading up on it now.. Thanks for your help!

**Comment by spacebuilder2020** [comment]
NP.  Sorry about the forge essentials plots, but forge essentials is really not designed for Greif Prevention and most of those features don't work super well.  Mytown2 has much better plot support cause it use a sql database to keep track of claims instead of just a text file.  I still plan to take a look at plots when I have time cause they should at least be usable.

**Comment by bardkin** [comment]
Mytown2 is pretty much overkill and for some reason I have to log out and back in before some changes become active. 
So I bounced back to the FE version to try a few other things and have a lot of issues-
#1 I do not have anything called FEData\permissions.json. Do I have to create that?
#2 several of the tutorial examples dont work whatsoever- setting spawn- i even did /setworldspawn but when I do /spawn it says unknown error.
#3 adding permissions- like I added fe.teleport.back to both user and group but it gives erroneous errors- cool down (even after the user reloggs) no permissions..  
#4 it wont let me look at the group the player belongs to. I cut and pasted right from the wiki (substituting the player name ) and it pukes out errors.
#5 when I create an entry or exit message- "Welcome to Spawn Town!" and then enter the zone, it says "Welcome, to, Spawn, Town!,"


Is there a set of generic basic permission files with dependency info I can look at? The Wiki is turning out to be very unhelpful. Many of the /p command statements are not working at all- for instance I want to look at the players user group, but it wont let me.

**Comment by spacebuilder2020** [comment]
The FEData is in the world folder

**Comment by bardkin** [comment]
yes, I have the FEData folder, but there is no .json file in there, nor is there a "permissions.json anywhere on my system that I can find.

Better if I can look at some kind of generic working code and dependency info.. I can understand real working examples better if I know what I have vrs whats might be missing and can create them or modify them to fit my needs.

**Comment by spacebuilder2020** [comment]
@bardkin Please update to 1.4.4.  It is a waste of both of our time to try and find out what is wrong with an old version of forge essentials

**Comment by bardkin** [comment]
So I updated and still its the same ol shenanigans. Area entry / exit messages seldom work if they ever do, If I do /zone info Spawn I get 0,0,0. and it's no where near those coordinates.
You cant turn mob griefing off in specific zones like the wiki says it can. Honestly, I think the update may be worse!
For whatever reason, the /area creates a named area but doesn't save or attach the boundaries. (why does the command even exist?)

It's like being caught between a rock and a hard place. I WANT to use a forge server, the mods are way better. but if I have to go back to spigot-
 just to have a spawn and a town protection area    
then I'll also have to figure out a good quest plugin to use, which they don't really have.

ridiculous.

**Comment by spacebuilder2020** [comment]
Ok.  I will try to take a look at it.  There are a lot of issues with plots as I recall.

**Comment by seanamor** [comment]
@Bardkin I know its not a permanent solution, but until these issues are patched, you should really look into sponge/forge. I have been using it for about 6 months now, and there are a couple plugins that help with world protection/perms Fox Guard, Fox Core, and Permissions Manager.

When there are true releases for 1.8.9 I will likely be moving to Forge Essentials to minimize the amount of mods/plugins

**Comment by spacebuilder2020** [comment]
Also if you are on 1.7.10, mytown2 is a excellent solution.  I will try to fix plots eventually, but mods like mytown2 that are dedicated to preventing grief will always be better.

**Comment by bardkin** [comment]
I will go to the Mytown2 area- I cant seem to get it to work at all. When I try to use any of the commands it says I don't have permission. I changed the .cfg to show using FE but got the same results.
Again, I will post issue there. 

I will also look into sponge, not at all familiar with it though.

**Comment by olee** [comment]
@bardkin to delete the area you should have used just `5` or `Plot1` (though name only works when you are in the same world as a player and not from server console).
